### ISSUE TITLE
 QUEUE=message php   vendor/bin/resque

### ISSUE THREAD

**Issue by starfishs** [chrisboulton/php-resque#373]
Hi,all 
in command line ,I run ` QUEUE=message php   vendor/bin/resque`;then I push many task into queue .but only one queue was execute.others also in queue.not execute.
anyone can help me ?

**Comment by danhunsaker** [comment]
This repo is deprecated. You'll want to try the one at [resque/php-resque](https://github.com/resque/php-resque).

**Comment by starfishs** [comment]
yeah.I know,the repo [resque/php-resque.](https://github.com/resque/php-resque) also has the same question.

**Comment by danhunsaker** [comment]
I'm not seeing it there?

**Comment by starfishs** [comment]
ok,  I report the issue in [ resque/php-resque.](https://github.com/resque/php-resque)
